galite-tests/src/test/kotlin/org/kopi/galite/tests/dsl/FormDSLTests.kt

```diff
@@ -163,16 +168,17 @@ class FormDSLTests: VApplicationTestBase() {
   fun `test block indexes`() {
     val form = MultipleBlockForm()
     val formModel = form.model
+    val index = form.block2.vBlock.getFieldIndex(form.block2.CenterId.vField)
 
-    assertEquals(form.block2.vBlock.getFieldIndex(form.block2.CenterId.vField), formModel.blocks.get(1).getFieldIndex(formModel.blocks.get(1).idField))
+    assertEquals(index, formModel.blocks[1].getFieldIndex(formModel.blocks[1].idField))
   }
```

Comment:
True, the `getFieldIndex` function is used for both expected and actual values, but from different contexts. the expected one is called from a form>block>VBlock, while the current is called from form>VDictionnaryFrom>VBlock. Currently, the indexes are not accessible from FormBlock as the `indices` is a private attribut. @hfazai  what do you advise?

Reply:
`indices` should be public as in some cases it can be used by user to iterate over indices of a form. Then you can test if model is getting indices defined by DSL.

Reply:
I was doing the wrong tests here, getFieldIndex just returns the field index (order) not the field index (table index). Now with making indices public we can verify if the assigned index for different fields is the same or not.